packages/lesswrong/components/posts/PostSubmit.tsx

```diff
@@ -5,6 +5,8 @@ import Button from '@material-ui/core/Button';
 import classNames from 'classnames';
 import { useCurrentUser } from "../common/withUser";
 import { useTracking } from "../../lib/analyticsEvents";
+import { getSiteUrl} from "../../lib/vulcan-lib";
```

Comment:
Merge imports, and there's probably an editor-config tweak or something to make lints visible sooner.

Reply:
Thanks. My editor actually did warn me, but only subtly.